backend/api/parse_pdf.py

```
@@ -338,19 +342,366 @@ async def parse_with_llamaparse(file: UploadFile, session_id: str = None) -> str
+                    personality_traits=traits,
+                    image_url=figure.get("image_url") or figure.get("imageUrl"),
+                    created_at=datetime.utcnow(),
+                    updated_at=datetime.utcnow()
+                )
+                db.add(persona)
+                existing_persona_names.add(persona_name)  # Track to avoid duplicates in same batch
+        
+        db.commit()
+        debug_log(f"[BACKGROUND_SAVE] Successfully saved autofill data for scenario {scenario_id}")
+        
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)